### PR TITLE
feature/v1-070/open course on mobile

### PR DESCRIPTION
**Pull request by Julia-Maliutsina**
Open detailed course on mobile devices by click on course item.

### CODE REVIEW

File: frontend/src/pages/MyCourses/MyCoursesContainer.tsx [hunk 1]
```diff
@@ -6,8 +6,15 @@ import MyCoursesList from './MyCoursesList';
 
 const MyCoursesContainer: React.FC = () => {
   const { data, isLoading } = useGetMyCourses();
+  const windowWidth = window.innerWidth < 425 ? 'small' : 'large';
```
Comment (NikitaKhadnevich) [inline frontend/src/pages/MyCourses/MyCoursesContainer.tsx:10]: Lets replace magic number to const

**Review by NikitaKhadnevich — CHANGES_REQUESTED** [review]

File: frontend/src/pages/MyCourses/MyCoursesContainer.tsx (hunk 1 above)
Comment (Julia-Maliutsina) [reply frontend/src/pages/MyCourses/MyCoursesContainer.tsx:12]: fixed

**Review by NikitaKhadnevich — DISMISSED** [review]

**Review by hauteballe — APPROVED** [review]

**Review by hauteballe — APPROVED** [review]

**Review by NikitaKhadnevich — APPROVED** [review]

File: frontend/src/pages/DetailedCourse/styled.ts [hunk 2]
```diff
@@ -4,6 +4,7 @@ import { Grid, Typography, Button as TextButton } from '@mui/material';
 
 import theme from 'themeSettings';
 import { Button } from 'components/Button';
+import { Link } from 'react-router-dom';
```
Comment (dapodev) [inline frontend/src/pages/DetailedCourse/styled.ts:7]: Please, move this line to other lib imports

**Review by dapodev — CHANGES_REQUESTED** [review]

File: frontend/src/pages/DetailedCourse/styled.ts (hunk 2 above)
Comment (Julia-Maliutsina) [reply frontend/src/pages/DetailedCourse/styled.ts]: Done

**Pull request merged**